seatunnel-engine/seatunnel-engine-common/src/main/java/org/apache/seatunnel/engine/common/config/SeaTunnelClientConfig.java

```diff
@@ -23,6 +23,15 @@
 
 public class SeaTunnelClientConfig extends ClientConfig {
 
+    public static final InitActionParallelism<Integer, Integer, Integer> INIT_ACTION_PARALLELISM = (actionParallelism, executionParallelism, upstreamParallelism) -> {
```

Comment:
Such changes are unreasonable.
You can add a utility method to get the correct parallelism when parsing Actions.